### DISCUSSION TITLE
Accept array by argument

### DISCUSSION THREAD

**Discussion by lucasantoniooficial** [rebing/graphql-laravel#1074] [General]
Have you already sent an array using the arguments?

I need to do this, I consume GraphQL in NextJS using axios.

**Comment by mfn** [comment]
I'm not sure what you're asking for, but when defining `args()`, you can require lists for example:
```php
    public function args(): array
    {
        return [
            'ids' => [
                'type' => Type::nonNull(Type::listOf(Type::nonNull(Type::id()))),
            ],
        ];
    }
```

If you mean something else, you've to be more specific.
